doc/cephfs/administration.rst

```diff
@@ -395,3 +395,14 @@ This removes a rank from the failed set.
 
 This command resets the file system state to defaults, except for the name and
 pools. Non-zero ranks are saved in the stopped set.
+
+
+::
+
+    fs new <file system name> <metadata pool name> <data pool name> --fs_id <fs_id> --force
+
+This command creates a file system with a specific ID. You may want to do this
```

Comment:
I've added this command in the advanced section since we don't want folks to use do this during regular operation mode.